lib/etl/transform/transform_epub.rb

```diff
@@ -0,0 +1,67 @@
+require 'epub/parser'
+
+class TransformEpub
```

Comment:
Eh, the answer is "yes" but maybe "not yet"? I kind of wanted something in place first.

I'll make an issue for this.